lib/chromedriver.js

```diff
@@ -32,7 +33,7 @@ class Chromedriver extends events.EventEmitter {
   async initChromedriverPath () {
     if (this.executableVerified) return;
     let binPath = this.chromedriver || (await getChromedriverBinaryPath());
-    if (!(await exists(binPath))) {
+    if (!(await fs.exists(binPath))) {
```

Comment:
Did we? I didn't know. I'll go through for it (though this is not intended to be a full clean up, just attempting to get the package to install correctly).